docs/vsphere-golden-image.md

```diff
@@ -68,17 +68,18 @@ the network interface and creates the `tools.conf` file.
 
 Install and configure the OpenSSH Server on the Windows virtual machine with automatic startup and key based-authentication.
 See [Microsoft documentation](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse)
-to install using PowerShell.
+to install using PowerShell. Given we are setting up SSH access for an administrator, please follow the
+[Microsoft's documentation](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_keymanagement#administrative-user)
```

Comment:
`Microsoft`. Yes this was an error I introduced but you should proof read 😉

Reply:
Fixed

/cc @alinaryan